ui/pages/login.tsx

```diff
@@ -103,13 +106,20 @@ const Login: NextPage = () => {
         }
         return "password";
       };
+      const method = getMethod();
+      if (
+        method === "password" &&
+        !(values as UpdateLoginFlowWithPasswordMethod).password
+      ) {
+        (values as UpdateLoginFlowWithPasswordMethod).password = "";
+      }
```

Comment:
may I suggest we refactor the boolean condition and the action to a function with a "speaking" name? 
So it's clear to anyone what is happening and why

Reply:
added a couple of const to make this easy to read.